core/src/sql/tokenizer.rs

```diff
@@ -21,6 +22,7 @@ impl Display for Tokenizer {
 			Self::Camel => "CAMEL",
 			Self::Class => "CLASS",
 			Self::Punct => "PUNCT",
+			Self::Jieba => "Jieba",
```

Comment:
Is it okay that this is not in capital letters @emmanuel-keller?

Reply:
It is okay as tokenizer names are case insensitive in SurrealQL, but it is still better to be aligned with the current convention. @RustLoverTheCoder
```suggestion
			Self::Jieba => "JIEBA",
```